bindings/core/src/method/wallet_operation.rs

```diff
@@ -18,28 +18,18 @@ use iota_sdk::{
         payload::transaction::TransactionId,
     },
     wallet::{
-        account::{
-            ConsolidationParams, CreateAccountParams, CreateNativeTokenParams, FilterOptions, MintNftParams,
-            OutputParams, OutputsToClaim, SyncOptions, TransactionOptions,
-        },
-        SendNativeTokensParams, SendNftParams, SendParams,
+        ConsolidationParams, CreateAccountParams, CreateNativeTokenParams, FilterOptions, MintNftParams, OutputParams,
+        OutputsToClaim, SendNativeTokensParams, SendNftParams, SendParams, SyncOptions, TransactionOptions,
     },
     U256,
 };
 use serde::{Deserialize, Serialize};
 
-/// Each public account method.
+/// Each public wallet operation method.
 #[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(tag = "name", content = "data", rename_all = "camelCase")]
 #[non_exhaustive]
-pub enum AccountMethod {
-    /// List addresses.
-    /// Expected response: [`Addresses`](crate::Response::Addresses)
-    Addresses,
-    /// Returns only addresses of the account with unspent outputs
-    /// Expected response:
-    /// [`AddressesWithUnspentOutputs`](crate::Response::AddressesWithUnspentOutputs)
-    AddressesWithUnspentOutputs,
+pub enum WalletOperationMethod {
```

Comment:
Why the separation still?

Reply:
several reasons actually ... mostly because I wasn't actually sure whether shouldn't be handled in a separate PR (since this one is already too big for my taste, so it might be a bit less to check as a reviewer), then I feel like there's still a difference between `methods` managing the wallet (`backup`, etc) and the methods that are interacting with an IOTA network. The `WalletMethod`s aren't exactly related implied by the protocol, while the `WalletOperationMethod`s kinda are (working with outputs, addresses, participation, voting, ...). I am open to discussing merging them if that is what the team wants.

Reply:
Separation might make sense for a better overview, but don't have a strong opinion, would also be fine with merging
But don't think we should care that much about the bindings in this PR

Reply:
I prefer the separation! so 👍